src/data/landing/guia-de-python.es.yml

```diff
@@ -0,0 +1,138 @@
+meta_info:
+  slug: guia-de-python # <--- URL 4geeksacademy.com/us/landing/example
+  # This will show on social media when shared
+  title: "Guía rápida de python para principiantes"
+  description: "Python un lenguaje de programación que está revolucionando el mercado siendo usado por empresas como: Netflix, Spotify, Amazon y muchos más."
+  image: "/images/bg/remote.jpeg"
+  keywords: "Aprende a programar, 4Geeks academy, aprende programacion, aprende ingenieria informatica"
+  template: "landing_a" # <--- choose your landing template
+  redirects: ["/landing/guia-de-python"] # <--- URL redirections to avoid 404 errors
+  # Set a particular course or location to the landing page
+  # this UTM fields will be also included in the CRM contact
+  utm_course: ["full-stack"]
+  utm_location: madrid-spain
+  automation: soft #optional, this is the default value
```

Comment:
Create a new tests that validates that this academy `utm_location` has available the automation and the tag on the beathecode API.

GET /marketing/academy/automation/<automation_slug>
GET /marketing/academy/tag/<tag_slug>

Reply:
olvidate de esta validacion porque require que tengas un token y ya quedamos en que no ibamos a usar token.